### PR TITLE
HLR: Remove deprecated v1 code

### PR DESCRIPTION
**Pull request by Mottie**
[Original Ticket #41973](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41973)

URL of fix: [Higher-Level Review introduction page](https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction)

# Expected behavior

With the HLR v2 flag set to 100%, all users should only see v2 questions in the wizard (`/start` page) and within the form. If a save-in-progress has v1 data stored, it should migrate into the v2 data format automatically.

## Current behavior

HLR v1 code has been deprecated. The main reason for this PR is to remove all v1 code to ensure that the Lighthouse v1 endpoint isn't called. It still is calling v1 endpoints, even after investigating why this is still the case.

## Your fix

Strip out v1 feature flags from app. Remove v1 related components, widgets, pages, configurations and tests. I did update the migration code to fix the representative phone number (missed before), and in so doing, this would force a new call to the migration code. But, I added a method to force code migration just-in-case. We really don't want v1 data to be submitted.

## How has this been tested?

- Removed v1 unit tests
- Updated v2 tests
- E2E testing

## Screenshots

No visual difference for v2

## Acceptance criteria
- [x] All v1 code removed from app
- [x] All tests passing

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/appeals/996/config/submitForm.js [hunk 1]
```diff
@@ -21,7 +19,7 @@ const submitForm = (form, formConfig) => {
     ? formConfig.transformForSubmit(formConfig, form)
     : transformForSubmit(formConfig, form);
 
-  const version = form.data.hlrV2 ? '1' : '0';
+  const version = '1';
```
Comment (scottymeyers) [inline src/applications/appeals/996/config/submitForm.js:22]: Same thing here, in my eyes, one less var the better.

**Review by scottymeyers — CHANGES_REQUESTED** [review]
@Mottie awesome, you did so much work here! if you have a moment today could you just walk me through the experience quickly?

File: src/applications/appeals/996/migrations/01-lighthouse-v2-updates.js [hunk 2]
```diff
@@ -2,6 +2,37 @@ import { isVersion1Data } from '../utils/helpers';
 
 const unaffectedPages = ['/contact-information', '/veteran-information'];
 
+export const forceV2Migration = data => {
+  const formData = {
+    ...data,
+    veteran: {
+      ...data.veteran,
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/996/migrations/01-lighthouse-v2-updates.js:9]: It does, but I wanted to make sure that the `veteran` object was defined... but I guess `...data.veteran` would throw an error if undefined. LOL

File: src/applications/appeals/996/actions/index.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ export const FETCH_CONTESTABLE_ISSUES_FAILED =
 
 export const getContestableIssues = props => {
   const benefitType = props?.benefitType || DEFAULT_BENEFIT_TYPE;
-  const apiOptions = { apiVersion: props.hlrV2 ? 'v1' : 'v0' };
+  const apiOptions = { apiVersion: 'v1' };
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/996/actions/index.js:17]: Good call, thanks!

File: src/applications/appeals/996/config/submitForm.js (hunk 1 above)
Comment (Mottie) [reply src/applications/appeals/996/config/submitForm.js:22]: Yep, I'll change this too!

File: src/applications/appeals/996/migrations/01-lighthouse-v2-updates.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/996/migrations/01-lighthouse-v2-updates.js:9]: It's all good, I think I'll just go with `const formData = { ...data };` LOL

**Review by scottymeyers — APPROVED** [review]

**Pull request merged**